pkgs/by-name/te/ternimal/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustc
+}:
+stdenv.mkDerivation rec {
+  pname = "ternimal";
+  version = "unstable-2017-12-31";
+
+  src = fetchFromGitHub {
+    owner = "p-e-w";
+    repo = pname;
```

```suggestion
    repo = "ternimal";
```

Technically, it is unrelated to `pname`, which is rather arbitrary.

---

more info here https://github.com/NixOS/nixpkgs/issues/277994